src/main/java/org/springframework/data/elasticsearch/annotations/Field.java

```diff
@@ -29,13 +31,30 @@
  * @author Jonathan Yan
  * @author Jakub Vavrik
  * @author Kevin Leturc
+ * @author Peter-Josef Meisch
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 @Documented
 @Inherited
 public @interface Field {
 
+	/**
+	 * The name to be used to store the field inside the document. Alias for {@link #name()}.
+	 *
+	 * @return an empty {@link String} by default.
+	 */
+	@AliasFor("name")
+	String value() default "";
+
+	/**
+	 * The name to be used to store the field inside the document. Alias for {@link #value()}.
+	 *
+	 * @return an empty {@link String} by default.
```

Comment:
Same here.

Reply:
the default value must be a constant, so we need an empty String here and cannot use null. To use the alias for the two attributes named _value_ and _name_ was proposed by Mark, this is the same as is used in spring-data-mongodb.

Reply:
Don't get me wrong, I just want you to adjust the comment to indicate that the field name is optional and by default is set to the property name and not to an empty string.

Reply:
have it adjusted yesterday with the other changes; I can't see the changes on the Github _conversation_ tab for this PR, but on the _Files changed_ tab I can see it.

Reply:
Check out https://github.com/spring-projects/spring-framework/blob/d035593562a1dd39c213143b4c44574051a3fb76/spring-web/src/main/java/org/springframework/web/bind/annotation/RequestParam.java#L66-L77 and https://github.com/spring-projects/spring-framework/blob/44ec6e003762b8bf571b0735746249f4697440e0/spring-web/src/main/java/org/springframework/web/bind/annotation/ResponseStatus.java#L65 for how Spring Framework documents aliased annotation properties.

Reply:
@mp911de thanks, that's exactly what I meant, that we don't need the `@return`.
@sothawoIt Sorry I didn't explain it better., would be nice if you could readjust it :)

Reply:
hope I got it right this time.

Reply:
Yes you got it ;)